data/json/recipes/tools/tools_electronic.json

```
@@ -26,7 +26,7 @@
     "decomp_learn": 2,
     "autolearn": [ [ "mechanics", 3 ], [ "electronics", 2 ] ],
     "book_learn": [ [ "manual_mechanics", 3 ], [ "textbook_mechanics", 2 ], [ "welding_book", 1 ] ],
-    "proficiencies": [ { "proficiency": "prof_welding_basic" }, { "proficiency": "prof_welding" } ],
+    "proficiencies": [ { "proficiency": "prof_welding_basic" } ],
```

Proficiencies don't need to be all-or-nothing. I think the correct solution here is to keep both proficiencies but reduce their impact on time and failure, as well as how much of the proficiencies you will gain from crafting this.

Remember that in design terms, having a proficiency should be thought of as a bonus. Rather than thinking "do I need to know how to weld to make this", think "would being a skilled welder make me better at making this". I find that is easier to grasp. I think the answer is yes, being a skilled welder would help you make a good welding tool, but not as much as it presently does. Feel free to reduce the multipliers, especially for the full welding proficiency.

```suggestion
    "proficiencies": [ { "proficiency": "prof_welding_basic" }, { "proficiency": "prof_welding", "time_multiplier": 1.1, "fail_multiplier": 1.1, "learning_time_multiplier": 0.25 } ],
```